src/main/java/target/HttpTarget.java

```diff
@@ -156,7 +156,7 @@ public CompletableFuture<Object> call(final ConsumerRecord<String, String> recor
                 connectionFailureDelayFactor
             )
             .withMaxRetries(Config.CONNECTION_FAILURE_RETRY_POLICY_MAX_RETRIES)
-            .handle(ConnectException.class)
```

Comment:
We probably need to remove the `handle` from the other policy

Reply:
I think we should keep it. All it does it gives us more retries

Reply:
i am not sure what happens when policies clash so I think we should be careful here

Reply:
the last one on the list (connectionFailureRetryPolicy) acts first and when it's done the other one (retryPolicy) does